File: Assets/L_Lua/Test/TestModuleBase.lua

TestViewSystem = require("Test.TestViewSystem")
TestViewController = require("Test.TestViewController")
require("Test.TestTools")

local M = class("TestModuleBase",ModuleBaseSystems)

TestModule = Context:New()

function M:ViewBindContoller()
	self._view_bind_controller[TestViewSystem] = TestViewController
end


function M:GetRequirePaths()
	local paths = 
	{
		"Test.TestTools",
		"Test.TestModuleBase",
		"Test.TestViewController",
		"Test.TestViewSystem"
	};
	return paths
end

return M;

File: Assets/Script/TestModule/TestModuleBase.cs
/****************************************************
 *  Copyright © 2018-2020 #AUTHORNAME#. All rights reserved.
 *------------------------------------------------------------------------
 *  文件：TestModuleBase.cs
 *  作者：Lonely
 *  日期：Created by #CreateTime#
 *  功能：Nothing
*****************************************************/

using Entitas;
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public enum ModuleID
{
	TestModule = 1
}

public enum TEST_MODDULE_EVENT
{
	SHOW_TEST_VIEW = 1
}

public class TestModuleBase : ModuleBaseSystems
{


    protected override void ViewBindContoller()
    {
        _view_bind_controller.Add(typeof(TestViewSystem), typeof(TestViewController));
    }



}
